linux/step05_centos7_nginx.sh

```diff
@@ -1,15 +1,19 @@
 #!/bin/bash
 
+#start-copy
+cp setup_omero_nginx.sh ~omero
+#end-copy
+
 #start-install
 yum -y --enablerepo=cr install nginx
```

Comment:
also this is not consistent with C6 where we manually install from nginx website. Is that expected?